http-client/build.gradle

```diff
@@ -1,6 +1,7 @@
 description = 'Http client library for http-server, provides an API for client-server interactions'
 
 dependencies {
+    implementation 'com.google.code.gson:gson:2.8.5'
```

Comment:
`http-client` has a dependency on this library, but it was previously obtained through a transitive dependency.  Gradle 5.0 apparently changes how transitive dependencies are exposed downstream, and thus we are now required to depend upon it explicitly.